auxx/src/Plugin.hs

```diff
@@ -60,20 +67,45 @@ auxxPlugin AuxxOptions{..} = \case
             w (addLogging sa)
 
 runWalletCmd :: (HasConfigurations, HasCompileInfo) => Text -> Worker AuxxMode
-runWalletCmd str sa = do
-    let strs = T.splitOn "," str
-    for_ strs $ \scmd -> do
-        let mcmd = parseCommand scmd
-        case mcmd of
-            Left err   -> putStrLn err
-            Right cmd' -> runCmd cmd' putText sa
-    putText "Command execution finished"
-    putText " " -- for exit by SIGPIPE
+runWalletCmd line sendActions = do
+    let commandProcs = createCommandProcs printAction sendActions
+    expr <- eitherToThrow $ Lang.parse line
+    value <- eitherToThrow =<< Lang.evaluate commandProcs expr
+    withValueText printAction value
+    printAction "Command execution finished"
+    printAction " " -- for exit by SIGPIPE
     liftIO $ hFlush stdout
 #if !(defined(mingw32_HOST_OS))
     delay $ sec 3
     liftIO $ exitImmediately ExitSuccess
 #endif
+  where
+    printAction = putText
+
+withValueText :: Monad m => (Text -> m ()) -> Lang.Value -> m ()
```

Comment:
Why did you do it this way instead of defining `instance Buildable Value` and simply using `printAction value`? It would like more conventional to me.
Is it just to conveniently  format `ValueList`? I guess it should be a big problem to format a list of values separated by spaces :/
And btw there is `listBuilder` in `Serokell.Util.Text` :)
(dinosaur)

Reply:
it was supposed to be a dirty and quick solution for printing :) ideally I want printed values to be valid language expressions (so one could copy REPL output and paste it back as input), and this requires a formatter for `Expr` (it's on my TODO list)

Reply:
Okay, can you create an issue about all the things you are going to do (write comments, implement a formatter for `Expr`, etc.)?

Reply:
Yes, I have them in orgmode at the moment. But I'd rather create separate issues, not one

Reply:
Okay, I will approve this PR after you create them.

Reply:
Created (see YT)